DependencyInjection/Configuration.php

```diff
@@ -71,7 +72,7 @@ public function getConfigTreeBuilder()
                     ->fixXmlConfig('container_type', 'container_types')
                     ->children()
                         ->scalarNode('enabled')->defaultValue('%kernel.debug%')->end()
-                        ->scalarNode('template')->defaultValue('SonataBlockBundle:Profiler:block.html.twig')->end()
+                        ->scalarNode('template')->defaultValue(version_compare(Kernel::VERSION, '2.7', '<') ? 'SonataBlockBundle:Profiler:block.html.twig' : 'SonataBlockBundle:Profiler:block_v3.html.twig')->end()
```

Comment:
You can use the `profiler_markup_version` variable directly on Twig.

More info: https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.8.md#webprofiler